custom-elements/reactions/AriaMixin-string-attributes.html

```diff
@@ -47,7 +47,6 @@
 testReflectAttribute('ariaPosInSet', 'aria-posinset', 'foo', 'bar', 'ariaPosInSet on Element');
 testReflectAttribute('ariaPressed', 'aria-pressed', 'foo', 'bar', 'ariaPressed on Element');
 testReflectAttribute('ariaReadOnly', 'aria-readonly', 'foo', 'bar', 'ariaReadOnly on Element');
-testReflectAttribute('ariaRelevant', 'aria-relevant', 'foo', 'bar', 'ariaRelevant on Element');
```

Comment:
Ditto here. These are alphabetical lists, so they should include all the known properties, even if some aren't testable yet.

We used a similar pattern in [html-aam/roles.html](https://github.com/web-platform-tests/wpt/blob/master/html-aam/roles.html) for completeness and pointers to avoid duplication of tests or issues.

Reply:
```diff
- testReflectAttribute('ariaRelevant', 'aria-relevant', 'foo', 'bar', 'ariaRelevant on Element');
+ // ariaRelevant blocked by https://github.com/w3c/aria/issue/1267
```

Reply:
"they should include all the known properties"

How do you define "known properties"? Looking at published specs, I can find nothing that would give me knowledge of `ariaRelevant`.

Did you mean instead "specified and proposed-for-the-future properties"?